@here/harp-mapview/lib/MapView.ts

```diff
@@ -1193,6 +1213,13 @@ export class MapView extends THREE.EventDispatcher {
         return this.canvas.height;
     }
 
+    /**
```

Comment:
Not relevant.

Reply:
Relevant in an accompanying internal file that estimates memory usage. The memory used on the GPU depends on this setting.